src/Microsoft.AspNetCore.Blazor.Razor.Extensions/BlazorMetadata.cs

```diff
@@ -15,7 +15,7 @@ public static class Bind
         {
             public static readonly string RuntimeName = "Blazor.None";
 
-            public readonly static string TagHelperKind = "Blazor.Bind-0.1";
```

Comment:
I don't think there's any value in versioning these things. I copied this design from Razor, but it predates some other (better) decisions we made about versioning there.